em_workflows/config.py

```diff
@@ -62,43 +64,37 @@ class Config:
     mrc2tif_loc = os.environ.get("MRC2TIF_LOC", "/opt/rml/imod/bin/mrc2tif")
     newstack_loc = os.environ.get("NEWSTACK_LOC", "/opt/rml/imod/bin/newstack")
 
-    # environment where the app gets run - used for share selection
-    env_to_share = {
-        "dev": "RMLEMHedwigDev",
-        "qa": "RMLEMHedwigQA",
-        "prod": "RMLEMHedwigProd",
-    }
-
     # All settings moved to respective constants file
     # fibsem_input_exts = ["TIFF", "tiff", "TIF", "tif"]
 
     SLURM_EXECUTOR = DaskExecutor(cluster_class=SLURM_exec)
     user = os.environ["USER"]
     tmp_dir = f"/gs1/Scratch/{user}_scratch/"
-    mount_point = "/mnt/ai-fas12/"
 
-    @staticmethod
-    def _share_name(env: str) -> str:
-        """
-        gets the path of the location of input based on environment
-        """
-        val = Config.env_to_share.get(env)
-        if not val:
-            raise ValueError(
-                f"Environment {env} not in valid environments: \
-                    {Config.env_to_share.keys()}"
+    @classmethod
+    def mount_point(cls, share_name: str) -> str:
+        try:
+            share_enum = FileShareEnum[share_name]
+        except KeyError as e:
+            prefect.context.logger.info(
+                f"{share_name} is a bad Share name which is not under consideration yet."
             )
-        return val
+            raise e
+        # TODO change the return values as necessary
+        mapper = {
+            FileShareEnum.RMLSOHedwigDev: f"/mnt/ai-fas12-so/{share_name}",
+            FileShareEnum.RMLSOHedwigQA: f"/mnt/ai-fas12-so/{share_name}",
+            FileShareEnum.RMLSOHedwigProd: f"/mnt/ai-fas12-so/{share_name}",
+        }
+        return mapper.get(share_enum, f"/mnt/ai-fas12/{share_name}")
```

Comment:
We use a mapper over the environment. The default mount_point is `mnt/ai-fas12/RMLEMHedwig...`.